nemo/collections/tts/models/fastpitch.py

```diff
@@ -612,11 +612,24 @@ def list_available_models(cls) -> 'List[PretrainedModelInfo]':
         )
         list_of_models.append(model)
 
-        # zh, single speaker, 22050Hz, SFSpeech Bilingual Chinese/English dataset
+        # zh, single female speaker, 22050Hz, SFSpeech Bilingual Chinese/English dataset, initial model.
         model = PretrainedModelInfo(
-            pretrained_model_name="tts_zh_fastpitch_sfspeech",
+            pretrained_model_name="tts_zh_fastpitch_sfspeech_r1_14",
             location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_zh_fastpitch_hifigan_sfspeech/versions/1.14.0/files/tts_zh_fastpitch_sfspeech.nemo",
-            description="This model is trained on a single female speaker in SFSpeech Bilingual Chinese/English dataset sampled at 22050Hz and can be used to generate female Mandarin Chinese voices.",
+            description="This model is trained on a single female speaker in SFSpeech Bilingual Chinese/English dataset"
+            " sampled at 22050Hz and can be used to generate female Mandarin Chinese voices.",
+            class_=cls,
+        )
+        list_of_models.append(model)
+
+        # zh, single female speaker, 22050Hz, SFSpeech Bilingual Chinese/English dataset, improved model using richer
+        # dict and jieba word segmenter for polyphone disambiguation.
```

Comment:
Any reason to keep the previous model version here as opposed to keeping the latest version as tag `tts_zh_fastpitch_sfspeech`? Users can still find the older versions corresponding to previous releases on NGC and download manually.

Reply:
In keeping with what we've done previously (and avoiding cluttering up the table since they are essentially the "same type" of model) I would prefer overriding. So when a user loads the `tts_zh_fastpitch_sfspeech` checkpoint they will always be guaranteed the latest checkpoint, and if for some reason they really need a previous version they can still find it.

Reply:
sounds good. Will revise.